docs/en/sql-reference/sql-statements/Data Definition/CREATE TABLE.md

```diff
@@ -296,9 +296,11 @@ Syntax:
         ```
         PROPERTIES (
             "storage_medium" = "[SSD|HDD]",
+            ["storage_cold_medium" = "[HDD|S3]"],
+            ["remote_storage_resource" = "xxx"],
```

Comment:
just use storage_resource= xxx , if it is defined then it implies that the cold storage medium type is s3 or hdd.

Reply:
`remote_storage_resource` must be used with `storage_cold_medium`.  Standalone use is not supported.
There is a check in `PropertyAnalyzer`.

Reply:
I don't think we need the conception of remote_storage_resource, if there is remote_storage_resource, there maybe a local_storage_resource.
And we could create a table on S3 or HDFS directly not in cold and hot scenario but in decouple compute and storage scenario.